pandas/core/groupby.py

```diff
@@ -1691,6 +1691,53 @@ def tail(self, n=5):
         mask = self._cumcount_array(ascending=False) < n
         return self._selected_obj[mask]
 
+    def pipe(self, func, *args, **kwargs):
+        """ Apply a function with arguments to this GroupBy object
+
+        .. versionadded:: 0.21.0
```

Comment:
instead of mostly repeated doc strings in both places, you can template _pipe and use and Appender/Substitution

Reply:
I would propose to not do this here. The dosctrings are sufficiently different to just make it a lot harder to develop when trying to merge them in a single one with a lot of substituted parts

Reply:
ok thats fine